shared/js/dataDriver.js

```diff
@@ -75,6 +75,8 @@ loop.DataDriver = function() {
         throw new Error("Missing option dispatcher");
       }
 
+      this._retryTimeout = this.INITIAL_RETRY_TIMEOUT;
```

Comment:
Since the code in connectToRoom resetting this was incorrect, I added the initialization in the constructor again, and reset the timeout in the "open" event handler. I think that's the right thing to do.